### ISSUE TITLE
cheat engine wont setup 

### ISSUE THREAD

**Issue by am74791111** [cheat-engine/cheat-engine#1133]
when i try to set up cheat engine it will just close  ?

**Comment by cheat-engine** [comment]
Uninstall badlion and restart (not shut down,  actual restart)

and check your anti virus

**Comment by am74791111** [comment]
for euro truck?

**Comment by am74791111** [comment]
i turned of anti virus but it still wont let me set up

**Comment by am74791111** [comment]
i give up

**Comment by cheat-engine** [comment]
execute `driverquery /v > bla.txt` in a command prompt and post the bla.txt file here

**Comment by nikitaivantsov** [comment]
[bla.txt](https://github.com/cheat-engine/cheat-engine/files/4177139/bla.txt)
I cannot install it either. I never had badlion or Karpersky on my system.
Setup just quietly exits after several seconds. There are 2 errors in Event Viewer

First
Faulting application name: CheatEngine70.tmp, version: 51.1052.0.0, time stamp: 0x57051f89
Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000
Exception code: 0xc00001a5
Fault offset: 0x070c6d2f
Faulting process id: 0xdcc
Faulting application start time: 0x01d5df62bd8946e7
Faulting application path: R:\Temp\is-S4L7F.tmp\CheatEngine70.tmp
Faulting module path: unknown
Report Id: e3802cfa-d726-4715-8ca4-c2fdfcc4c088
Faulting package full name: 
Faulting package-relative application ID: 

Second
Faulting application name: CheatEngine70.tmp, version: 51.1052.0.0, time stamp: 0x57051f89
Faulting module name: KERNELBASE.dll, version: 10.0.18362.628, time stamp: 0x54734dee
Exception code: 0x0eedfade
Fault offset: 0x00113db2
Faulting process id: 0xdcc
Faulting application start time: 0x01d5df62bd8946e7
Faulting application path: R:\Temp\is-S4L7F.tmp\CheatEngine70.tmp
Faulting module path: C:\WINDOWS\System32\KERNELBASE.dll
Report Id: 609ab530-8298-480a-856e-228044690ce6
Faulting package full name: 
Faulting package-relative application ID:

**Comment by GauthierPLM** [comment]
I have the same issue and can confirm the error message @nikitaivantsov posted. Here is mine (sorry for the french):

```
Nom de l’application défaillante CheatEngine70.tmp, version : 51.1052.0.0, horodatage : 0x57051f89
Nom du module défaillant : KERNELBASE.dll, version : 10.0.18362.628, horodatage : 0x54734dee
Code d’exception : 0x0eedfade
Décalage d’erreur : 0x00113db2
ID du processus défaillant : 0x1798
Heure de début de l’application défaillante : 0x01d5e01b18c4f1c4
Chemin d’accès de l’application défaillante : C:\Users\***\AppData\Local\Temp\is-8BJFQ.tmp\CheatEngine70.tmp
Chemin d’accès du module défaillant: C:\WINDOWS\System32\KERNELBASE.dll
ID de rapport : 7f3b06de-c997-4e0d-85b3-536221a34622
Nom complet du package défaillant : 
ID de l’application relative au package défaillant :
```

>

**Comment by cheat-engine** [comment]
Do you block the internet connection it tries to establish?

**Comment by nikitaivantsov** [comment]
Yes, all connections except whitelisted are blocked on my PC. Temporarily disabling firewall solved the issue.

**Comment by GauthierPLM** [comment]
I use Adguard, that might block the connection. But an installer should not require an internet connection and should not crash if the said connection is blocked or not available.

As I mentionned above, only version 7 seems affected by this issue.

**Comment by HeIp** [comment]
For anyone experiencing the same problem that did not solve the issue try this : https://www.cheatengine.org/download/CheatEngine7.0_MissingSetup.rar

**Comment by GauthierPLM** [comment]
@HeIp Has it been removed? I get a 403 error when I try to download from your link.

**Comment by HeIp** [comment]
@GauthierPLM Surprisingly it has been removed. @cheat-engine  Is there a reason for this ?

**Comment by cheat-engine** [comment]
it's not removed,  it's still at https://cheatengine.org/downloads.php click the 2nd 'here'

**Comment by HeIp** [comment]
You get a 403 - Forbidden error @cheat-engine

**Comment by cheat-engine** [comment]
Click the link with the left mouse button while on that website,  don't copy it

**Comment by fade2gray** [comment]
I was also getting the 403 error whether I clicked the 2nd 'here' on the download page or the link above provided by Help, but the error has now sorted itself out :/

![CE403](https://user-images.githubusercontent.com/1332009/74430932-c905ab00-4e55-11ea-84e8-b8b390cb3cff.JPG)

**Comment by HeIp** [comment]
It's working now @fade2gray & @GauthierPLM
